### PR TITLE
Update to ungoogled-chromium 121.0.6167.85

### PR DESCRIPTION
**Pull request by Cubik65536**
Notes:

- A submodule bump is made.
- A patch refresh is made.
- A patch `ungoogled-chromium/macos/disable-rust-qr-code-generator.patch` is added to revert the Rust QR code generator shipped with [[5064208]](https://chromium-review.googlesource.com/c/chromium/src/+/5064208).

---

Builds and runs fine locally.

image

### CODE REVIEW

**Comment by PF4Public** [comment]
Do we need these checks in PRs?

**Comment by Cubik65536** [comment]
> Do we need these checks in PRs? I mean attempting to build.

We don't, otherwise we'll actually just let CI do the exact same thing twice.

**Comment by PF4Public** [comment]
Maybe we should exclude building on PRs then?

**Comment by Cubik65536** [comment]
> Maybe we should exclude building on PRs then?

We could, I kept it when writing the new action script because it was there and sometime it was just skipped.

**Comment by PF4Public** [comment]
Would you like to disable it from building in another PR, so that we could merge this one?

**Comment by Cubik65536** [comment]
> Would you like to disable it from building in another PR, so that we could merge this one?

Ok I will make the change.

**Comment by Cubik65536** [comment]
> Would you like to disable it from building in another PR, so that we could merge this one?

done

**Review by PF4Public — APPROVED** [review]

**Comment by PF4Public** [comment]
Thanks!

**Comment by networkException** [comment]
@Cubik65536 I think you said you'd be willing to maintain the repo yourself already, if thats still the case I'd invite you and grant you access. You can always contact me for questions of course and because you'd be the only maintainer for now PRs will also require a core team member approval currently

**Comment by Cubik65536** [comment]
> @Cubik65536 I think you said you'd be willing to maintain the repo yourself already, if thats still the case I'd invite you and grant you access.

Yes I'm still interested in being a maintainer for the project :)

> You can always contact me for questions of course and because you'd be the only maintainer for now PRs will also require a core team member approval currently

No problem!

**Comment by networkException** [comment]
Welcome!

I guess the best way to try your new push privileges is to create the tag for this version :)

**Comment by Cubik65536** [comment]
> Welcome!
> 
> I guess the best way to try your new push privileges is to create the tag for this version :)

https://github.com/ungoogled-software/ungoogled-chromium-macos/tree/121.0.6167.85 🎉

Thanks!

**Comment by PF4Public** [comment]
> to create the tag

Shouldn't they be created via workflows?

**Comment by networkException** [comment]
they're not created via workflows in any repo of the org afaik

**Comment by Cubik65536** [comment]
> > to create the tag
> 
> Shouldn't they be created via workflows?

A tag creation (push to tag) it what triggers the workflow that build **and release** the binaries.

**Pull request merged**
